tests/qgis-projects/tests/qgis_info.qgs

```diff
@@ -0,0 +1,201 @@
+<qgis projectname="" version="3.10.13-A Coruña">
```

Comment:
@drillinP could you change file names. `qgis_info` doesn't tell what those files permit to test.